stylesheets/utilities.css

```diff
@@ -0,0 +1,67 @@
+/* padding */
+
+/* 1rem */
+.p-b-1{
+    padding-bottom: 1rem;
+}
+
+.p-l-1{
+    padding-bottom: 1rem;
+}
+
+.p-r-1{
+    padding-bottom: 1rem;
+}
+
+.p-1{
+    padding: 1rem;
+}
+
+/* 2rem */
+.p-b-2{
+    padding-bottom: 1rem;
+}
+
+.p-l-2{
+    padding-bottom: 1rem;
+}
+
+.p-r-2{
+    padding-bottom: 1rem;
+}
+
+.p-2{
+    padding: 1rem;
+}
```

Comment:
fix the padding size, i think its supposed to be 2 rem

Reply:
fixed it.